hardware_enablement/kmm-release-notes.adoc

```diff
@@ -315,3 +315,86 @@ For more information, see xref:../hardware_enablement/kmm-kernel-module-manageme
 // TELCODOCS-2390
 // MGMT-21050
 If you are running KMM-hub version 2.3.0 or earlier and you are not running KMM, the upgrade to KMM-hub 2.4.0 is not reliable. Instead, you must upgrade to KMM-hub 2.4.1. KMM is not affected by this issue. For more information, see link:https://access.redhat.com/errata/RHEA-2025:10778[RHEA-2025:10778 - Product Enhancement Advisory].
+
+[id="kmm-2-5-RN"]
+== Release notes for Kernel Module Management Operator 2.5
+=== New features and enhancements
+
+// MGMT-22178
+* The Kernel Module Management Operator (KMMO) 2.5 provides a `version.ready` label to indicate that the new version of the kernel module is loaded and is ready to use. For more information, see xref:../hardware_enablement/kmm-kernel-module-management.adoc#kmm-using-intree-modules_kernel-module-management-operator[].
+
+=== Notable technical changes
+
+// MGMT-21754
```

Comment:
This task doesn't contain the `doc-RN-needed` label and shouldn't be included in the release-notes.

Reply:
Removed.